lib/api/src/modules/refs.ts

```diff
@@ -180,10 +183,14 @@ export const init: ModuleFn = ({ store, provider, fullAPI }) => {
               credentials,
               cache: 'no-cache',
             })
-          ),
+          ).then((r: SetRefData) => (r?.error?.message === 'Failed to fetch' ? {} : r)),
         ]);
 
-        Object.assign(loadedData, { ...stories, ...metadata });
+        Object.assign(loadedData, { ...stories, ...storiesWithAuth, ...metadata });
+
+        if (storiesWithAuth && !storiesWithAuth.loginUrl) {
+          delete loadedData.loginUrl;
+        }
```

Comment:
You won't need to do this is if you are careful not to include results that didn't work (i.e. non-credentialed requests to a server that requires them).